include/xtensor/xsort.hpp

```diff
@@ -13,15 +13,22 @@
 #include <algorithm>
 #include <utility>
 
-#include "xarray.hpp"
-#include "xeval.hpp"
-#include "xslice.hpp"  // for xnone
-#include "xmanipulation.hpp"
-#include "xtensor.hpp"
-#include "xtensor_config.hpp"
+#include "xtensor/xarray.hpp"
+#include "xtensor/xeval.hpp"
+#include "xtensor/xslice.hpp"  // for xnone
+#include "xtensor/xmanipulation.hpp"
+#include "xtensor/xtensor.hpp"
+#include "xtensor/xtensor_config.hpp"
```

Comment:
```suggestion
#include "xarray.hpp"
#include "xeval.hpp"
#include "xslice.hpp"  // for xnone
#include "xmanipulation.hpp"
#include "xtensor.hpp"
#include "xtensor_config.hpp"
```

1. Does not belong in the PR.
2. I don't think this is the right thing to do?

Reply:
1. Fair enough
2. It's debatable, I've always used the latter so since it was not consisitent in xtensor, I've thought it was a good direction. Reverting.